compiler/tests/lowering/fixtures/base_tuple_lowered.fe

```diff
@@ -0,0 +1,68 @@
+struct tuple_u256_bool:
+    item0: u256
+    item1: bool
+
+struct tuple_u256_bool_u8_address:
+    item0: u256
+    item1: bool
+    item2: u8
+    item3: address
+
+struct tuple_bool_bool:
+    item0: bool
+    item1: bool
+
+struct tuple_bool_address:
+    item0: bool
+    item1: address
+
+struct tuple_address_address:
+    item0: address
+    item1: address
+
+struct tuple_address_address_u16_i32_bool:
+    item0: address
+    item1: address
+    item2: u16
+    item3: i32
+    item4: bool
+
+contract Foo:
+    my_tuple_field: tuple_bool_address
+
+    event MyEvent:
+        my_tuple: tuple_bool_bool
+        my_other_tuple: tuple_address_address
+
+    pub def bar(my_num: u256, my_bool: bool) -> tuple_u256_bool:
+        return tuple_u256_bool(item0=my_num, item1=my_bool)
+
+    pub def baz() -> tuple_u256_bool_u8_address:
+        return tuple_u256_bool_u8_address(
+            item0=999999,
+            item1=false,
+            item2=u8(42),
+            item3=address(26)
+        )
+
+    pub def bing():
+        foo: tuple_address_address_u16_i32_bool = tuple_address_address_u16_i32_bool(
+            item0=address(0),
```

Comment:
Random fact, C# also exposes tuple items as `item0`, `item1` etc (just with a capitalized `I`) so we are in good company.

Reply:
Nice! Correct decision for sure :laughing: